helm/cert-manager-app/values.schema.json

```diff
@@ -215,7 +218,21 @@
                                     "type": "object",
                                     "properties": {
                                         "token": {
-                                            "type": ["null", "string"]
+                                            "type": "string"
```

Comment:
regenerating the schema file has removed the two `null` types - i would suggest that this is fine given that they are instead quoted as empty strings in the values file.